### PR TITLE
Feature/1677 geo location for api v2

### PR DESCRIPTION
**Pull request by kzangeli**
Fixes part of the issue #1677.

The coords for polygons has changed in v2.
New rules: last point must be equal to first point, and the number of points must be >= 4.
These two rules are implemented but *ifdeffed out*. Perhaps this should be fixed within this PR.
Also, there is a comment or two about more detailed 'details' in errors.
Both these (two rules + details) give errors in functests and are disabled in this PR.

### CODE REVIEW

File: test/valgrind/valgrindTestSuite.sh [hunk 1]
```diff
@@ -156,7 +156,7 @@ dryrun=off
 leakTest=off
 dryLeaks=off
 fromIx=0
-ixLists=""
+ixList=""
```
Comment (fgalan) [inline test/valgrind/valgrindTestSuite.sh]: Interesting fix... valgrind suite was broken before fixing this?

Comment (kzangeli) [inline test/valgrind/valgrindTestSuite.sh]: My guess is that the variable ixList was empty by default ...
Perhaps all shell variables are, and these initializations are not necessary ... ?

Comment (fgalan) [inline test/valgrind/valgrindTestSuite.sh]: NTC

**Comment by fgalan** [comment]
Speed-LGTM (this PR is actually part of an ongoing work in #1677 to be finished y me)

**Pull request merged**